main_app/templates/item/detail.html

```diff
@@ -14,10 +14,7 @@ <h1>Item Details</h1>
         <p>There are only <span style="color: red">{{ item.quantity_current }}</span> remaining. Please restock.</p>
         {% endif %}
     </div>
-    <div class="card-action">
-        <a href="{% url 'inventory_update' item.id %}">Edit</a>
```

Comment:
Are we sure we only want the options to edit/delete on the home page for the list view?